route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManagerV5.cs

```diff
@@ -1192,6 +1213,18 @@ public DataObjectRoute[] GetRoutes(RouteParametersQuery routeParameters, out Res
                 HttpMethodType.Get);
         }
 
+        public DataTypes.DataObjectRoute[] GetRoutesByFilter(RouteFilterParameters routeFilterParameters,
```

Comment:
Do you plan to introduce async version for introduced APIs?

Reply:
Yes, I'll add async version as well.

Reply:
I've seen added async API for GetRoutesByFilter. The same I think should be done for all new APIs. I just avoid putting comment on each and every new API.

Reply:
Ok, I'll do it.